cli/app.go

```diff
@@ -105,8 +105,16 @@ var App = &cli.App{
 		{
 			Name:    "dump",
 			Aliases: []string{"d"},
-			Usage:   "dumps a previously compiled dictionary to its original ODXML",
-			Action:  dump,
+			Usage:   "dumps a previously compiled dictionary (defaults to original ODXML)",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:    "format",
+					Aliases: []string{"f"},
+					Usage:   "Specific output format of the dump (ODXML or SQL)",
```

Comment:
I would omit "(ODXML or SQL)", as we could mention that in the docs (just in an effort to keep these CLI-based help messages concise). However, on a broader note, I don't think we actually have a way to list all the possible output formats. Could be helpful to maybe:

1. Error if the user gives an invalid format (not sure if this is already configured), and list the possible output formats there.
2. Provide a separate command for listing all the formats.

What do you think?

Reply:
I think this is a great idea. Currently, if you don't match any of the sql dialects in the above commands and you do something like `go run odict.go d -f fakeFormatLol examples/example1.odict examples/output.sql` it will dump the odict to ODXML to the specified file.

Adding the invalid message + list of valid formats seems like the best option for UX!

Reply:
Done as per the comment left below!